docs/changelog/100033.yaml

```diff
@@ -0,0 +1,9 @@
+pr: 100033
+summary: "[Behavioral Analytics] Analytics collections use DLM instead of ILM for\
```

Comment:
```suggestion
summary: "[Behavioral Analytics] Analytics collections use DSL instead of ILM for\
```

Reply:
We renamed DLM to `Data stream lifecycle` ( https://www.elastic.co/guide/en/elasticsearch/reference/master/data-stream-lifecycle.html )